parsers/to-style-dictionary/README.md

```diff
@@ -0,0 +1,457 @@
+# To Style Dictionary
+
+## Description
+
+Format design tokens to create a theme compatible with the [Style Dictionary specification](https://amzn.github.io/style-dictionary/#/).
```

Comment:
Format design tokens to create a working [Style Dictionary configuration](https://amzn.github.io/style-dictionary/#/).